src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -740,6 +736,48 @@ public CodecType AudioCodecType
             }
         }
 
+        /// <summary>
+        /// Gets or sets the default codec type of the video decoder.
```

Comment:
I'm not sure what "default" means here, why it is added. But it's copied from AudioCodecType property, so let's keep it. Still, do you know in what sense it is a "default" codec? Why not just "the (current) codec of decoder"?

Reply:
It means, when user get `VideoCodecType` property for the first time without setting `VideoCodectype`, user will get `default` video codec type set by system. But after set `VideoCodecType`, user will get video codec type set by user. 
So I'll remove the word `default` from here.